src/Features/Core/Portable/PullMemberUp/AbstractMembersPullerService.cs

```diff
@@ -25,14 +25,20 @@ namespace Microsoft.CodeAnalysis.CodeRefactorings.PullMemberUp
     internal abstract class AbstractMembersPullerService<TUsingOrAliasSyntax> : IMembersPullerService
         where TUsingOrAliasSyntax : SyntaxNode
     {
-        protected abstract SyntaxNode EnsureLeadingTriviaBeforeFirstMember(SyntaxNode root, SyntaxTriviaList trivia);
-
-        protected abstract SyntaxTriviaList GetFirstMemberTrivia(SyntaxNode root);
+        /// <summary>
+        /// Gets the members of <paramref name="root"/>, so we can save/re-add its leading trivia.
+        /// </summary>
+        protected abstract SyntaxList<SyntaxNode> GetMembers(SyntaxNode root);
```

Comment:
this is misleading as from teh impl i know that 'root' doesn't need to be the root of the tree.